README.md

```diff
@@ -1,13 +1,13 @@
 # Restore xcodebuild Bitrise Build Cache
 
-[![Step changelog](https://shields.io/github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache/releases)
+[![Step changelog](https://shields.io/github/v/release/bitrise-steplib/bitrise-step-restore-xcodebuild-cache?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache/releases)
 
-Restores the DerivedData folder and the related metadata from Bitrise Build Cache to speed up subsequent builds.
+Restores the DerivedData folder and related metadata used by xcodebuild. Uses the Bitrise Build Cache infrastructure.
 
 <details>
 <summary>Description</summary>
 
-This step is the pair of the [save-xcodebuild-cache step](https://github.com/bitrise-steplib/bitrise-step-save-xcodebuild-cache) and can be used to restore xcodebuild cache archives from the Bitrise Build Cache. 
+This steps restores the DerivedData folder and the related metadata required to speed up subsequent builds.
```

Comment:
```suggestion
This step restores the DerivedData folder and the related metadata required to speed up subsequent builds.
```

Reply:
The README is generated from the code (`bitrise run generate_readme`) - so we should fix it there.